### PR TITLE
add app icon using cmake and rc file.

### PR DESCRIPTION
**Pull request by dsm**
Hi, this PR, add app icon using cmake and rc file and also adds app icon to qucs.exe

image
image

### CODE REVIEW

File: qucs/qucs.cpp [hunk 1]
```diff
@@ -265,7 +265,7 @@ void QucsApp::initView()
   // set application icon
   // APPLE sets the QApplication icon with Info.plist
 #ifndef __APPLE__
-  setWindowIcon (QPixmap(":/bitmaps/big.qucs.xpm"));
+  //setWindowIcon (QPixmap(QString(":/bitmaps/hicolor/scalable/apps/qucs.svg")));
```
Comment (ra3xdh) [inline qucs/qucs.cpp:268]: Why this line is commented?

**Comment by ra3xdh** [comment]
Why the line 268 in qucs.cpp is commented?

**Comment by dsm** [comment]
cmake changes changed icon in windows and line 268 override if we comment out. if you accept qucs.svg everywhere I comment out.

**Comment by ra3xdh** [comment]
Yes, I am accepting `qucs.svg`

**Comment by dsm** [comment]
I comment out line 268

**Comment by ra3xdh** [comment]
Merged.

**Pull request merged**